### DISCUSSION TITLE
Unable to connect to github

### DISCUSSION THREAD

**Discussion by lie16** [community/community#52577] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Maybe this is a noob question but I had a problem with connecting to GitHub. Below are the case:
I just move my server recently into new public IP and provider, but somehow I'm unable to connect to GitHub. The connection ended at 104.44.230.62. I don't know what can cause these, could anyone help me?

**Comment by sahana-github** [comment]
Hi @lie16 maybe this might help you ...
Check your firewall settings Make sure that your firewall is not blocking the connection to GitHub. GitHub uses SSH (port 22) and HTTPS (port 443) protocols. Ensure that these ports are open in your firewall.

**Reply by lie16** [reply]
It's open, and when I run tracert the connection ended at 104.44.230.62 (already reach msn.net). I couldn't even visit github.com when I try another git services I could only open gitlab.com

**Comment by sahana-github** [comment]
@lie16 maybe try using a different internet connection If possible, try connecting to GitHub from a different internet connection, such as a mobile hotspot or a different Wi-Fi network. This will help you determine if the issue is with your network or with GitHub.

**Reply by lie16** [reply]
At home, I could pull normally, but at the server, the connection always ended at msn. I'm suspecting that the server's public IP is being blacklisted, but don't know how to prove it

**Comment by sahana-github** [comment]
@lie16 Check your IP address against GitHub's firewall, GitHub uses a firewall to protect against malicious traffic. If your IP address has been flagged by their firewall, you may be unable to connect to their servers. You can check if your IP address is flagged by visiting https://api.github.com/meta and looking for your IP address in the "hooks" section. If your IP address is flagged, you can contact GitHub's support team to resolve the issue.

**Reply by airtower-luna** [reply]
The IP addresses listed in the meta API response are addresses owned by GitHub, and sorted by what they're used for. The "hooks" list would be for sending webhooks (so if you want to receive webhooks, you'd need to allow traffic *from* those addresses in your firewall).

**Comment by sahana-github** [comment]
Hope This Helps 😐

**Reply by lie16** [reply]
Thank's for the help. I'll contact them hopefully get some answer

**Comment by airtower-luna** [comment]
Here's the official troubleshooting guide: https://docs.github.com/en/get-started/using-github/troubleshooting-connectivity-problems

If you make a report, best include the output of the debug site mentioned near the bottom of that article. :slightly_smiling_face:
